recipes/svjedi/meta.yaml

```diff
@@ -15,9 +15,9 @@ source:
 
 requirements:
   run:
-    - python>=3
+    - python >=3
     - numpy
-    - minimap2
+    - minimap2 2.17
```

Comment:
why is this pinning needed?

Reply:
Hello,
The minimap2 tool has changed considerably and our tool shows better results with this old version of minimap2. 
Lolita